shared/menubar/index.desktop.js

```diff
@@ -256,15 +270,12 @@ const BadgeIcon = ({
   )
 }
 
-MenubarRender.defaultProps = {
-  openToPrivate: true,
-  openWithMenuShowing: false,
-}
-
 const borderRadius = 4
 
 const stylesContainer = {
   ...globalStyles.flexBoxColumn,
+  width: '100%',
```

Comment:
This component is getting an extra spacing on the top now, maybe from this?

![image](https://user-images.githubusercontent.com/11968340/33579439-d4d95090-d917-11e7-89a4-23aaddd03f4b.png)

Reply:
good catch! turns out this was a teeny bit more complicated. see the last commit